versions/development/SPEC.md

```diff
@@ -698,6 +699,48 @@ Pair values can also be specified within the [workflow inputs JSON](https://gith
 }
 ```
 
+### Optional literals
+
+Any non-optional value can be wrapped into an optional value by assigning it to an optional variable:
+
+```WDL
+Int? maybe_integer = 5
+```
+
+The `None` literal is the value that an optional has when it is not defined.
+
+```WDL
+Int? maybe_integer = None
+# maybe_integer is not defined
+Int? maybe_numeral = 5
+Int certain_integer = 3
+# maybe_numeral is a defined optional
+is_defined(maybe_integer) # Evaluates to false
```

Comment:
I think the WDL function is `defined()`, not `is_defined()`. Also would it be clearer to write these lines as if they were inputs or outputs? As it is now they're WDL code fragments. e.g.
```suggestion
Boolean test1 = defined(maybe_integer) # Evaluates to false
```

Reply:
That is more clear indeed. I have added this